blocks/library/more/index.js

```diff
@@ -21,7 +21,10 @@ registerBlockType( 'core/more', {
 
 	useOnce: true,
 
-	className: false,
+	support: {
+		generatedClassname: false,
```

Comment:
The camel-casing of `className` is inconsistent here (`Classname` vs. `className`). I think we ought to call this `generatedClassName`.

Aside: I like `import classnames from 'classnames';` only because camel-casing in the case of NPM dependencies should be on the basis of dash-delimiters (i.e. if it were `class-names`).